main.go

```diff
@@ -47,7 +47,13 @@ func main() {
 }
 
 // keyCmd is used to generate a new private key.
-type keyCmd struct{}
+type keyCmd struct{
+	Args struct {
```

Comment:
The nested struct is not needed (other commands use the nested struct to collect positional args).